src/shared/install.c

```diff
@@ -1421,23 +1431,23 @@ static int unit_file_search(
                         return -ENOMEM;
 
                 r = unit_file_load_or_readlink(c, info, path, paths->root_dir, flags);
+
```

Comment:
We like to keep the action and the handling of the result nearby. This empty line is unnecessary.